api/src/org/labkey/api/data/ContainerManager.java

```diff
@@ -2585,7 +2585,7 @@ public static Container getMoveTargetContainer(@NotNull String entityType, @NotN
 
         if (!_targetContainer.hasPermission(user, InsertPermission.class))
         {
-            errors.reject(ERROR_GENERIC, "You do not have permission to move " + entityType + " to the target container: " + targetIdOrPath + ".");
+            errors.reject(ERROR_GENERIC, "You do not have permission to move rows to this target container: " + targetIdOrPath + ".");

```

Comment:
```suggestion
            errors.reject(ERROR_GENERIC, "You do not have permission to move rows to the target container: " + targetIdOrPath + ".");

```
Do we have the query name that we could pass in here for better user feedback ("permission to move rows from 'My Sample Type' to the target container: X")?

Reply:
Yes, added queryName to error message